flusher.go

```diff
@@ -721,16 +721,15 @@ func flushSpansDatadog(ctx context.Context, s *Server, nilTracer func() opentrac
 			parentID = 0
 		}
 
-		resource := span.Tags[trace.ResourceKey]
-		name := span.Tags[trace.NameKey]
+		resource := span.Tags["resource"]
```

Comment:
Could we leave this as a constant instead of a string literal? We can change the name to something more descriptive to its Datadog-specific purpose.